chartjs-java-model/src/main/java/software/xdev/chartjs/model/options/Plugins.java

```diff
@@ -41,6 +42,22 @@ public Plugins setTitle(final Title title)
 		this.title = title;
 		return this;
 	}
+
+	/**
+	 * @see #setSubtitle(Title)
+	 */
+        public Title getSubtitle() {
+          return this.subtitle;
+        }
+
+	/**
+	 * The subtitle configuration is passed into the options.subtitle namespace. The global options for the chart subtitle is
+	 * defined in Chart.defaults.global.subtitle.
+	 */
```

Comment:
Looks like your Javadoc formatting is off?

Reply:
I use Ctrl+Shift+F to format the code and javadoc, but it changes the format completely. So, I made it manually :) Is there a better way to do it?

Reply:
Yeah you did the right thing not reformatting the whole file i think fixing it by hand is fine.

Reply:
> I use Ctrl+Shift+F to format the code and javadoc, but it changes the format completely. So, I made it manually :) Is there a better way to do it?

Have a look at our [CONTRIBUTING.md](https://github.com/xdev-software/chartjs-java-model/blob/develop/CONTRIBUTING.md#developing). Once setup correctly your IntelliJ does everything for you.